Spigot-Server-Patches/0591-Fixed-EntityLiving-not-respecting-isCancelled.patch

```diff
@@ -0,0 +1,76 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Jordan Adams <jordan@jordanplayz158.me>
+Date: Sun, 18 Oct 2020 19:13:30 -0400
+Subject: [PATCH] Fixed EntityLiving not respecting isCancelled
+
+
+diff --git a/src/main/java/net/minecraft/server/EntityLiving.java b/src/main/java/net/minecraft/server/EntityLiving.java
+index 76185f042dd773f267dc7574116cec7bd8a28ca8..e20e7a51766fd5afdbe8ab9369980e12da5689d8 100644
+--- a/src/main/java/net/minecraft/server/EntityLiving.java
++++ b/src/main/java/net/minecraft/server/EntityLiving.java
+@@ -1,6 +1,7 @@
+ package net.minecraft.server;
+ 
+-import com.destroystokyo.paper.event.player.PlayerArmorChangeEvent; // Paper
```

Comment:
imports should not be changed, especially given how the only intended change of this patch is a diff from ``continue`` to ``break`` on one line.

Reply:
Yeah i'm honestly not sure why it messed with the imports, I never told the ide to so it must've been an automatic thing from IntelliJ, my bad.